### ISSUE TITLE
Focus state for switch

### ISSUE THREAD

**Issue by jgiardino** [patternfly/patternfly-next#1824]
Since the Switch is not a native html element (like a button or text input), but instead is a checkbox styled to look like a switch, we cannot use native browser styles for indicating focus.

For this issue, we should add styles that display on focus to indicate to the user when the switch has focus.

**Comment by srambach** [comment]
Per @mceledonia this should be styled in the universal focus state styling, see #790 
With specs:
- 8px padding from edges of object
- Fill: #06C at 15% opacity
- Border: 1px #06C at 100%

**Comment by srambach** [comment]
After discussion with @mceledonia, we'll go with a 2px #06C border, 8 px from the edge, but no fill.
